ACCESSIBILITY.md

```diff
@@ -0,0 +1,90 @@
+
+2. **Enabling accessibility of our shared community platforms**: Our community is dedicated to enabling access across the various platforms we communicate on, which extends not only to the accessibility practices mentioned above, but also to the language we employ to ensure a broad audience. Our [slack welcome guide](https://the-turing-way.netlify.app/community-handbook/communication-channels/slack-welcome-guide.html) delineates best practices we use on the platform.
+
+3. **Implementing accessible practices for community events**: As a community, we frequently organise events for a variety of purposes: for coworking, trainings, workshops, panels, conference talks, and more. We are committed to supplying and recording audio transcripts and closed captioning, uploading slides and/or presentation materials, and enabling broad access to our program content [as described in our community handbook](https://book.the-turing-way.org/community-handbook/accessibility/event-organising).
+
+### 3.0 How we work with Access-related Feedback
+
+Gathering access-related feedback is integral for ensuring that we can remain accessible as a community. We would like to offer a a few ways of sharing feedback with us, and are currently working to develop the following initiatives:
+
+1. **Feedback form**: Please submit access related comments to our [accessibility feedback form](https://forms.gle/Ngr2eUtQmf7aEeao6). All of the information in this form is processed anonymously, with the option of submitting contact information by choice. 
+2. **Drop-in sessions**: Throughout the calendar year, we would like to facilitate drop-in session to receive feedback about access related different parts of project. These may take place at our [Collaboration Cafe](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-collabcafe.html) or may be hosted separately. 
+3. **Email the Research Community Manager or project team directly directly**: You can always email the Research Community Manager directly at asteele@turing.ac.uk with direct feedback. Alternatively, you can also reach out to her on Slack, or to any of the [project delivery team members](https://github.com/the-turing-way/the-turing-way/blob/main/ways_of_working.md).
+4. **Attend the Accessibility Working Group meeting**: The Accessibility Working Group meetings may be a place where you can deliver feedback in a small group environment. These meetings are held on Zoom on the second Monday of each month at 17:00 UK time. Please join the #Accessibility channel on slack for more information.
+
+We recognise that many feedback forms and other forms for gathering feedback can often operate as "blackboxes" that receive information without taking actionable steps to address them.
```

Comment:
Suggested rephrasing:

We recognise that oftentimes feedback and evaluation forms operate as 'blackboxes' that receive information without taking actionable steps to address them. Knowing this, we are actively working to develop processes that enable us to implement feedback and to report transparently. These may include:

Reply:
Love this wording - thanks a lot @LelleDi !